### ISSUE TITLE
Don't work Git LFS for me

### ISSUE THREAD

**Issue by Kristinita** [github/git-lfs#1492]
## Purpose

I want to download large files in my repository on GitHub.

## Problem

I can not configure Git LFS. Now I can not make a push into my repository on GitHub.

## Steps to reproduce

I download and installed Git LFS to folder `E:\Git LFS`. I opened command prompt in folder `E:\wamp\www `, where my local repository. I add to folder `E:\wamp\www ` files `Сериал. Туристы 10 серия из 24 ( 2005 ). DVDRip. AVI..mp4` and `rsload.net.krita-3.0-x86.zip` with `mp4` and `zip` extensions. Size of these files exceeds 100 megabytes. (I understand that GitHub - not hosting for a these files, I just wanted to test the capabilities of Git LFS, and then delete these two files of GitHub.)

```
E:\wamp\www>git lfs track "*.zip"
  Tracking *.zip
```

```
E:\wamp\www>git lfs track "*.mp4"
  Tracking *.mp4
```

`git add .` →` git commit -m "Description"` →  but I can not make a push my changes.

```
E:\wamp\www>git push
  Counting objects: 28, done.
  Delta compression using up to 4 threads.
  Compressing objects: 100% (28/28), done.
  Writing objects: 100% (28/28), 298.44 MiB | 439.00 KiB/s, done.
  Total 28 (delta 11), reused 0 (delta 0)
  remote: Resolving deltas: 100% (11/11), completed with 3 local objects.
  remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
  remote: error: Trace: 0aa332480459045c5a774d19ca217c24
  remote: error: See http://git.io/iEPt8g for more information.
  remote: error: File rsload.net.krita-3.0-x86.zip is 100.34 MB; this exceeds GitHub's file size limit of 100.00 MB
  remote: error: File Сериал. Туристы 10 серия из 24 ( 2005 ). DVDRip. AVI..mp4 is 199.75 MB; this exceeds GitHub's file size limit of 100.00 MB
  To https://github.com/Kristinita/Kristinita.github.io.git
   ! [remote rejected] master -> master (pre-receive hook declined)
  error: failed to push some refs to 'https://github.com/Kristinita/Kristinita.github.io.git'
```

## System requirements

Windows 10 Education
git version 2.9.3.windows.1
git-lfs/1.4.1 (GitHub; windows 386; go 1.7; git 2070e4c)

Thanks.

**Comment by sinbad** [comment]
Had you previously committed these files to this repository *before* you did the `git lfs track` commands? It appears that your repository has history for `rsload.net.krita-3.0-x86.zip` and `Сериал. Туристы 10 серия из 24 ( 2005 ). DVDRip. AVI..mp4` where they were not tracked in LFS but embedded directly in git.

You should either start from a blank repository and track these files as your first action, or use a tool like [git-lfs-migrate](https://github.com/bozaro/git-lfs-migrate) to convert history to LFS.

**Comment by Kristinita** [comment]
How I solved my problem:

1. I remove `.git` folder and `.gitattributes` file from my local repository.
2. I move in other place my big files.
3. In folder of my repository I open command prompt.
4. `git init`, `git remote add origin https://github.com/Kristinita/Kristinita.github.io.git`
5. Since I could not make a push or a pull, I used the command `git reset --hard origin/master`.
6. `git lfs track "*.zip"`, 
   `git lfs track "*.mp4`"
7. I newly added large files in my repository.
8. `git add .`, `git commit -m "Git-LFS"`, `git push`. 

I successfully add my large files to remote repository. See my [commit](https://github.com/Kristinita/Kristinita.github.io/commit/ae30630a105f88c338c4789855884b7b04f4e20f). 

Thanks.
